messagequeue/chunkeroptions_test.go

```diff
@@ -0,0 +1,47 @@
+package messagequeue
+
+import (
+	"fmt"
+	"testing"
+
+	assert1 "github.com/stretchr/testify/assert"
+)
+
+func TestNewChunkerOptions(t *testing.T) {
+	assert := assert1.New(t)
+	options := NewChunkerOptions()
+	assert.NotNil(options)
+	assert.NoError(options.Validate())
+}
+
+func TestChunkerOptions_Valid(t *testing.T) {
+	assert := assert1.New(t)
+
+	options := NewChunkerOptions()
+
```

Comment:
should you validate that the options are set to the default values?

Reply:
I went down that path at first, but I think that the expectation is that the returned instance is 'Valid'. Because the defaults are not part of the public interface, the values on this valid instance are what is considered default. The defaults in constants would be more of a style/best practices which is tested by having the code be reviewed.